packages/core/src/components/select/select.tsx

```diff
@@ -50,7 +50,7 @@ export class AtomSelect {
   @Event() atomFocus!: EventEmitter<void>
 
   @Method()
-  setTagInSelectOptions() {
+  async setTagInSelectOptions() {
```

Comment:
do you know why it is complaining? I was seeing the code and I don't see any promise :thinking:

Reply:
@tassioFront stenci requires all methods to be asynchronous because its entire API is asynchronous. Even if there is no promise, we need to make them asynchronous. 😆 
I discovered this today while working on the tooltip. :haha:

Reply:
Exactly @tassioFront, it was causing a warning during the build 😆

Reply:
dammn, I didn't know that haha thanks!